docs/local_install_skaffold.md

```diff
@@ -108,81 +355,300 @@ Running the following command will do it for you (you'll be asked for your passw
+### Stop Substra
+
+In order to stop Substra, hit `ctrl + c` in each repository. On Ubuntu, if you want to stop the minikube Kubernetes cluster, you can use `minikube stop`.
+
+If you want to restart, you will just need to run again the `skaffold run` command in the 3 repositories.
+
+### Reset Substra
+
+You can reset your installation (if you've used `skaffold run`) with:
+
+```sh
+# run from each repository (hlf-k8s, substra-backend, substra-frontend)
+skaffold delete
+# or
+kubectl rm ns peer-1 peer-2 orderer
+# On Ubuntu, to remove all the Kubernetes cluster
+minikube delete
 ```
 
-## Login, password and urls
+## 4. Login, password and urls
+
+### Credentials and urls
 
 Once the platform is running, you can sign in to the two organizations using the default development credentials:
 
 org-1:
+
 - API url: `http://substra-backend.node-1.com`
 - Frontend url: `http://substra-frontend.node-1.com`
 - Username: `node-1`
 - Password: `p@$swr0d44`
 
 org-2:
+
 - API url: `http://substra-backend.node-2.com`
 - Frontend url: `http://substra-frontend.node-2.com`
 - Username: `node-2`
 - Password: `p@$swr0d45`
+
+You should find the credentials in the charts: `skaffold.yaml` files or in the `substra-backend/charts/substra-backend/values.yaml`.
+
+### Browser extension
+
+In order to use the backend pages (API) on your browser, you will need to install this extension that will send a special header containing a version:
+
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/modheader-firefox/)
+- [Chrome](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj)
+
+You will then need to:
+
+- Add `Request Header`: `text/html;version=0.0, */*; version=0.0`
+- Add `Filters` > `URL Pattern`: `http://susbtra-backend.node-1.com`
+- Add `Filters` > `URL Pattern`: `http://susbtra-backend.node-2.com`
+
+TODO: [KO] Or import this to the extension:
+
+```json
+[{"title":"Profile 1","hideComment":true,"headers":[   {"enabled":true,"name":"Accept","value":"text/html;version=0.0, */*; version=0.0","comment":""}],"respHeaders":[],"filters":[{"enabled":true,"type":"urls","urlRegex":"http://substra-backend.node-2.com"},{"enabled":true,"type":"urls","urlRegex":"http://susbtra-backend.node-1.com"}],"urlReplacements":[],"appendMode":false}]
+```
+
+See: <https://github.com/SubstraFoundation/substra-backend#testing-with-the-browsable-api>
+
+If you are running the `substra/examples`, you will need to edit the `config.json` as follows:
+
+Change from:
+
+```json
+{
+  "profile_name": "owkin",
+  "url": "http://substra-backend.owkin.xyz:8000",
+  "username": "substra",
+  "password": "p@$swr0d44"
+}
+```
+
+To:
+
+```json
+{
+  "profile_name": "owkin",
+  "url": "http://substra-backend.node-1.com:80",
+  "username": "node-1",
+  "password": "p@$swr0d44"
+}
+```
+
+See: <https://github.com/SubstraFoundation/substra/blob/master/docs/local_install_skaffold.md#login-password-and-urls>
+
+### Substra CLI config & login
```

Comment:
Maybe the docker-compose doc could link to this section for the part on the CLI and SDK ?